docs/source/user-guide/tabular/supported_models.rst

```diff
@@ -121,6 +121,15 @@ ROC Curve and the AUC metric) will not run.
 Running Deepchecks with Pre-computed Predictions
 ------------------------------------------------
 
+It is possible to pass pre-computed predictions to the checks, this can be useful for example when the model is not
```

Comment:
Not sure this additional text is required, this info is IMO covered in the text above

Reply:
Tend to agree. What is the added value of this paragraph?